universe/config.json

```diff
@@ -171,7 +190,7 @@
                     "description": "The containerizer used to deploy the Jenkins master.",
                     "type": "string",
                     "enum": ["MESOS", "DOCKER"],
-                    "default": "DOCKER"
```

Comment:
Why not leave it to DOCKER if that is what they set it to?
Does it not work with Docker and portworx changes?

Reply:
portworx volume changes are not working with containerizer DOCKER, so default set it to MESOS